test/bin/mdx-test/failure/in-toplevel/test-case.md.expected

```diff
@@ -1,2 +1,2 @@
-Error in the toplevel code block in test-case.md at line 4:
```

Comment:
Do you know why this was removed? Is there a good reason for it?

Same goes for the other occurences!

Reply:
This error message depends on the block value, which was previously built during evaluation:
```ocaml
let eval t =
  match t.header with
  | Some ("sh" | "bash") ->
      let value = cram t.contents in
      { t with value }
  | Some "ocaml" -> (
      match guess_ocaml_kind t with
      | `OCaml `Code -> { t with value = OCaml }
      | `OCaml `Toplevel ->
          let value = toplevel ~file:t.file ~line:t.line t.contents in
          { t with value }
      | `Other -> Fmt.failwith "Dead code. todo: remove" )
  | _ -> t
```
which decided it was a toplevel block, but now we see there is a `file` label so we decide it is an Include block, we can do something with the header of include blocks maybe

Reply:
Could you please include the kind of MDX block (not the header) that the error comes from when it's available?

I think it will help users figure out the origin of such issues, in particular they might have misunderstood how MDX would interpret the block.

Reply:
To clarif, I think the error should probably say something like `Error in the file include code block ...` or even eventually `Error in the OCaml file include code block ...`

Reply:
done